apps/labrinth/.env.local

```diff
@@ -25,6 +25,11 @@ MODERATION_SLACK_WEBHOOK=
 PUBLIC_DISCORD_WEBHOOK=
 CLOUDFLARE_INTEGRATION=false
 
+PAYOUT_ALERT_SLACK_WEBHOOK=none
+TREMENDOUS_BALANCE_ALERT_THRESHOLD=0
+PAYPAL_BALANCE_ALERT_THRESHOLD=0
+BREX_BALANCE_ALERT_THRESHOLD=0
```

Comment:
I think we should add these variables to the `.env.docker-compose` sample env file as well.